docs/DeveloperGuide.md

```diff
@@ -736,7 +742,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 <div style="page-break-after: always;"></div>
 
 ### Use cases
-
+[//]: # (@@author chiageng)
```

Comment:
I think I wrote this use case in v1.1....

Reply:
yup, resolved